src/main/groovy/com/blackducksoftware/integration/hub/detect/DetectConfiguration.groovy

```diff
@@ -305,6 +315,10 @@ class DetectConfiguration {
     @Value('${blackduck.hub.offline.mode:}')
     Boolean hubOfflineMode
 
+    @ValueDescription(description="If set to false we will not automatically resolve the ~ character to the user's home directory.", defaultValue="true", group=DetectConfiguration.GROUP_PATHS)
```

Comment:
I feel it's valuable to specify that we will only resolve paths that start with ~/ to paths that start with the user home directory, not that we will resolve all ~ in the path.